### PR TITLE
Add NEP Organization to NCR Silver Auth

### PR DESCRIPTION
**Pull request by Frouget**

### CODE REVIEW

File: pages/connector-authentication/NCRSilver-Authentication.md [hunk 1]
```diff
@@ -19,14 +19,15 @@ To set up an account, go to the [NCR website](https://developer.ncr.com/):
...
-You need a **Shared Key** and **Secret Key** in order to access the connector.
+You need a **Shared Key** and **Secret Key** in order to access the connector. You will also need a **NEP Organization** value.
 
 To get these from your account, go to the [NCR website](https://developer.ncr.com/):
 1.  Log in to your account.
 2.  Select your email at the top right corner of the window and select dashboard.
 3.  Select your application.
-4.  Select the **+** icon next to **Access Keys**. 
-5.  Make note of the **Shared Key** and **Secret Key**.
+4.  **NEP Organization** is located under Service User Name. It is the value after "acct:" and before the "@" symbol.
```
Comment (developingharry) [inline pages/connector-authentication/NCRSilver-Authentication.md:28]: 4.  **NEP Organization** is located under Service User Name. It is the value after "acct:" and before the "@" symbol.  

If the Service User Name is acct:**test-drive-399bcc60-9426-4e46-bf9c-76f3e6e5883e**@44423c74-6324-446b-97d2-e88538c3c926 for example, the **NEP Organization** you need is **test-drive-399bcc60-9426-4e46-bf9c-76f3e6e5883e**.

Comment (developingharry) [reply pages/connector-authentication/NCRSilver-Authentication.md:28]: Something like that?  Maybe formatted a bit better (the review editor is a bit naff).

Feel like there should be an example anyway.

**Review by developingharry — CHANGES_REQUESTED** [review]
See notes :)

File: pages/connector-authentication/NCRSilver-Authentication.md [hunk 2]
```diff
@@ -19,14 +19,16 @@ To set up an account, go to the [NCR website](https://developer.ncr.com/):
 
 ### Get authentication details
 
-You need a **Shared Key** and **Secret Key** in order to access the connector.
+You need a **Shared Key** and **Secret Key** in order to access the connector. You will also need a **NEP Organization** value.
```
Comment (erin-lw) [inline pages/connector-authentication/NCRSilver-Authentication.md:22]:
```suggestion
You need a **Shared Key** and **Secret Key** in order to access the connector. You also need a **NEP Organization** value.

```

File: pages/connector-authentication/NCRSilver-Authentication.md [hunk 3]
```diff
@@ -19,14 +19,16 @@ To set up an account, go to the [NCR website](https://developer.ncr.com/):
...
-You need a **Shared Key** and **Secret Key** in order to access the connector.
+You need a **Shared Key** and **Secret Key** in order to access the connector. You will also need a **NEP Organization** value.
 
 To get these from your account, go to the [NCR website](https://developer.ncr.com/):
 1.  Log in to your account.
 2.  Select your email at the top right corner of the window and select dashboard.
 3.  Select your application.
-4.  Select the **+** icon next to **Access Keys**. 
-5.  Make note of the **Shared Key** and **Secret Key**.
+4.  **NEP Organization** is located under Service User Name. It is the value after "acct:" and before the "@" symbol.
```
Comment (erin-lw) [inline pages/connector-authentication/NCRSilver-Authentication.md:28]:
```suggestion
4.  Make note of the **NEP Organization** value under **Service User Name**. This is the value after `acct:` and before the `@` symbol.

```

File: pages/connector-authentication/NCRSilver-Authentication.md [hunk 4]
```diff
@@ -19,14 +19,16 @@ To set up an account, go to the [NCR website](https://developer.ncr.com/):
...
+    Example: If the Service User Name is acct:test-drive-486bcc60-9426-4e46-bf9c-90l3e6e5883e@44423c74-6324-446b-97d2-e88538c3c926, the **NEP Organization** you need is test-drive-486bcc60-9426-4e46-bf9c-90l3e6e5883e.
```
Comment (erin-lw) [inline pages/connector-authentication/NCRSilver-Authentication.md:29]:
```suggestion
    *  For example, if the **Service User Name** is `acct:test-drive-486bcc60-9426-4e46-bf9c-90l3e6e5883e@44423c74-6324-446b-97d2-e88538c3c926`, the **NEP Organization** you need is `test-drive-486bcc60-9426-4e46-bf9c-90l3e6e5883e`.

```

**Pull request merged**